bcg6es-d1e110/cod-Zy765t_bcg6es-d1e110.xml

```diff
@@ -128,7 +128,7 @@
           <lb ed="#P" n="39"/>deles Christi seducere possunt, de quibus dictum est supra <ref xml:id="bcg6es-d1e110-Rd1e396">dist.  <!-- ggf. ref auf Dist 1 einfügen -->
             <!--566.xml-->
           <pb ed="#P" n="566"/>
-            <lb ed="#P" n="1"/>I.</ref> Vnde super illud <ref xml:id="bcg6es-d1e110-Rd1e406" corresp="#bcg6es-d1e110-Qd1e409">I. Corinth. 15.</ref> <quote xml:id="bcg6es-d1e110-Qd1e409" source="http://scta.info/resource/Icor15_33" synch="1-2">Nolite seduci;</quote> dicit <ref xml:id="bcg6es-d1e110-Rd1e412" corresp="#bcg6es-d1e110-Qd1e415">Glossa. <!-- Glosse--></ref> <quote xml:id="bcg6es-d1e110-Qd1e415">Per 
+            <lb ed="#P" n="1"/>I.</ref> Vnde super illud <ref xml:id="bcg6es-d1e110-Rd1e406" corresp="#bcg6es-d1e110-Qd1e409">I. Corinth. 15.</ref> <quote xml:id="bcg6es-d1e110-Qd1e409" source="http://scta.info/resource/Icor15_33" synch="1-2">Nolite seduci;</quote> dicit <ref xml:id="bcg6es-d1e110-Rd1e412" target="#bcg6es-d1e110-Qd1e415">Glossa.</ref> <quote xml:id="bcg6es-d1e110-Qd1e415">Per 
```

Comment:
This first part of this correct. Look how the `<ref>` for the bible verse contains the `@corresp` pointing to the `<quote>` that comes below. Do the same thing for the glossa `<ref>` that comes below. Use `@corresp` on the glossa `<ref>` to point to the glossa `<quote>` below.